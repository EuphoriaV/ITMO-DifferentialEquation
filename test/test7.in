3.53
((4.25 * 6.69) * (2.96 * 6.53))
3.11
2.11