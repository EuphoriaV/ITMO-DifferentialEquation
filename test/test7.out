0.0026932